plugins/sync.js

```diff
@@ -774,23 +851,32 @@ window.plugin.sync.setupCSS = function() {
   .appendTo("head");
 };
 
+////////////////////////////////////////////////////
+//// Main **********************************************************************************
 var setup = function() {
-  window.plugin.sync.logger = new plugin.sync.Logger({'logLimit':10, 'logUpdateCallback': plugin.sync.updateLog});
+  window.plugin.sync.logger = new plugin.sync.Logger(
+    {'logLimit':10, 'logUpdateCallback': plugin.sync.updateLog}
+  );
   window.plugin.sync.loadUUID();
   window.plugin.sync.setupCSS();
   window.plugin.sync.setupDialog();
-
-  window.plugin.sync.authorizer = new window.plugin.sync.Authorizer({
-    'authCallback': [plugin.sync.toggleAuthButton, plugin.sync.toggleDialogLink]
-  });
+// create an authorizer structure
+  window.plugin.sync.authorizer = new window.plugin.sync.Authorizer(
+    {
+    'authCallback': [plugin.sync.toggleAuthButton,          // functions to execute after 
+                     plugin.sync.toggleDialogLink           // successfull authorization
```

Comment:
another good example why this comment format is bad,
if you add another callback this comment get's messed up ... and it's currently hard to read if you're just looking for "toggleDialogLink"